beacon_node/network/src/router.rs

```diff
@@ -482,7 +482,10 @@ impl<T: BeaconChainTypes> Router<T> {
     ) {
         let request_id = match request_id {
             RequestId::Sync(sync_id) => match sync_id {
-                SyncId::SingleBlock { .. } | SyncId::ParentLookup { .. } => {
+                SyncId::SingleBlock { .. }
+                | SyncId::SingleBlob { .. }
+                | SyncId::ParentLookup { .. }
+                | SyncId::ParentLookupBlob { .. } => {
                     unreachable!("Block lookups do not request BBRange requests")
```

Comment:
Just to confirm my understanding, if we hit these unexpected `unreachable` code paths, it means there's a bug in our code on handling peer responses and there's no way we could recover even if we resend the request, hence the panic here?